tests/POW/test_POW.cpp

```diff
@@ -706,7 +706,7 @@ BOOST_AUTO_TEST_CASE(mining_and_verification)
     uint8_t difficultyToUse = 10;
     uint8_t blockToUse = 0;
     ethash_mining_result_t winning_result = POWClient.PoWMine(
-        blockToUse, difficultyToUse, rand1, rand2, ipAddr, pubKey, false);
+        blockToUse, difficultyToUse, rand1, rand2, ipAddr, pubKey, true);
```

Comment:
This is mining full but verifying light . Do we need to enable some more test ? @ckyang what are your thoughts ?

Reply:
The mining should always be full, otherwise it is too easy to get the result. The verify should use light because every client can verify the result quickly.
I agree we need to add more auto test case for it, but it will be after the GPU mining is ready.

Reply:
The issue is that if you verifyLight, it does not enforce full mining i.e. verifyLight would be true even if you mined light or mined full.

Reply:
Hi Kaustubh, sorry for delayed reply. This is a good question, i also don't have the answer yet. Need to discuss with the team how to enforce full mining in the future. @Gnnng @bb111189 @jiayaoqijia @Gnnng

Reply:
Yes, just trying to bring it to your attention

Reply:
@KaustubhShamshery @bb111189 @Gnnng @jiayaoqijia After i do some investigation and test, i find actually light dataset mine is much slower than full dataset mine. Because light dataset mine use less memory, every ethash access need to regenerate dag data, it takes a lot of time. I tested with PoW difficulty 15, light mine takes 19 seconds, full dataset mine only 9 seconds. So in the long run, people will not consider using light dataset mine.